modules/deploy-red-hat-openshift-container-storage.adoc

```diff
@@ -5,44 +5,44 @@
 
-|Instructions on deploying OpenShift Container Storage on Red Hat {product-title} VMWare vSphere clusters
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/deploying_openshift_container_storage_on_vmware_vsphere/index[Deploying OpenShift Container Storage 4.5 on VMWare vSphere]
+|Instructions on deploying {rh-storage} on Red Hat {product-title} VMWare vSphere clusters
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/deploying_openshift_container_storage_on_vmware_vsphere/index[Deploying OpenShift Data Foundation 4.5 on VMWare vSphere]
 
-|Instructions on deploying OpenShift Container Storage using Amazon Web Services for local or cloud storage
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/deploying_openshift_container_storage_using_amazon_web_services/index[Deploying OpenShift Container Storage 4.5 using Amazon Web Services]
+|Instructions on deploying {rh-storage} using Amazon Web Services for local or cloud storage
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/deploying_openshift_container_storage_using_amazon_web_services/index[Deploying OpenShift Data Foundation 4.5 using Amazon Web Services]
 
-|Instructions on deploying and managing OpenShift Container Storage on existing Red Hat {product-title} Google Cloud clusters
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/deploying_and_managing_openshift_container_storage_using_google_cloud/index[Deploying and managing OpenShift Container Storage 4.5 using Google Cloud]
+|Instructions on deploying and managing {rh-storage} on existing Red Hat {product-title} Google Cloud clusters
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/deploying_and_managing_openshift_container_storage_using_google_cloud/index[Deploying and managing OpenShift Data Foundation 4.5 using Google Cloud]
 
-|Instructions on deploying and managing OpenShift Container Storage on existing Red Hat {product-title} Azure clusters
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/deploying_and_managing_openshift_container_storage_using_microsoft_azure/index[Deploying and managing OpenShift Container Storage 4.5 using Microsoft Azure]
+|Instructions on deploying and managing {rh-storage} on existing Red Hat {product-title} Azure clusters
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/deploying_and_managing_openshift_container_storage_using_microsoft_azure/index[Deploying and managing OpenShift Data Foundation 4.5 using Microsoft Azure]
 
```

Comment:
@Erin-Donnelly Can you please add IBM Power and IBM Z. I've discussed this with @lpettyjo some time back. 
fyi: IBM Power went through a name change a few months ago they removed "Systems" from the name. 
```
|Instructions on deploying {rh-storage} to use local storage on IBM Power infrastructure
|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.8/html/deploying_openshift_container_storage_using_ibm_power_systems/[Deploying OpenShift Data Foundation 4.8 using IBM Power infrastructure]
```
```
|Instructions on deploying {rh-storage} to use local storage on IBM Z infrastructure
|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.8/html/deploying_openshift_container_storage_using_ibm_z_infrastructure/[Deploying OpenShift Data Foundation 4.8 using IBM Z infrastructure]
```

Is there any reason why you haven't changed to ODF 4.8? Maybe a new attribute `:rh-storage-version:` would be helpful for not having the need to update everywhere on a regular basis.

Reply:
@Erin-Donnelly Can you add IBM Power and IBM Z please.

Reply:
done here https://github.com/openshift/openshift-docs/pull/41858/commits/8ce1288f530195bad99e43e57a78c24b4365251f